backend/src/main/java/net/finman/mapper/ItemMapper.java

```diff
@@ -0,0 +1,21 @@
+package net.finman.mapper;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import org.springframework.jdbc.core.RowMapper;
+
+import net.finman.model.Item;
+
+public class ItemMapper implements RowMapper<Item> {
+
+    @Override
+    public Item mapRow(ResultSet rs, int rowNum) throws SQLException {
+        Item item = new Item();
+        item.setOwner(rs.getInt("owner"));
+        item.setName(rs.getString("name"));
+        item.setPrice(rs.getInt("price"));
+        item.setAmount(rs.getInt("amount"));
```

Comment:
There is no 'amount' field in in the table _Items_

Reply:
it retrieves from a joined table of InvoiceItems and Items, since we need both the amount and price attributes

Reply:
> 
> 
> There is no 'amount' field in in the table _Items_

it retrieves from a joined table of InvoiceItems and Items, since we need both the amount and price attributes